app/views/admin/users/index.csv.erb

```diff
@@ -2,14 +2,11 @@
 require 'csv'
 
 csv_output = CSV.generate(:col_sep => ",") do |csv|
-  columns = ['username', 'name', 'authentication_token', 'email', 'role', 'daily_requests', 'monthly_requests']
-
+  columns = %w[username name email role daily_requests monthly_requests]
```

Comment:
Note: Refactoring this whole view is in the backlog to implement pagination & filtering, so this CSV generation will be refactored in the future.